docs/commands.md

```diff
@@ -33,6 +33,17 @@ Execute `graphback` in your shell for more information
   Create databases resources based on your data model. Currently supports PostgreSQL and sqlite3.
 > **Note** - *Running `graphback db` drops your database and recreates it with tables and relationships.*
 
+## Update Database
+
+```sh
+graphback update-db
```

Comment:
Question - we can suggest update-db as well in generate output. Good idea?

Reply:
Yep good idea!

Reply:
Good idea!